doc/nrf/includes/wifi_credentials_shell.txt

```diff
@@ -0,0 +1,21 @@
+Configuring Wi-Fi access point credentials
+==========================================
+
+This sample uses the :ref:`lib_wifi_credentials` library to manage Wi-Fi credentials.
+Before the sample can connect to a Wi-Fi network, you must configure at least one credential set.
+
+Once your device has been flashed with this sample, connect to your device's UART interface and add credentials using the following command:
+
+.. parsed-literal::
+   :class: highlight
+
+   wifi_cred add *NetworkSSID* WPA2-PSK *NetworkPassword*
+
+Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password.
+Then either reboot the device or use the ``wifi_cred auto_connect`` command to manually trigger a connection attempt.
+
+From now on, these credentials will automatically be used when the configured network is reachable.
+
+Enabling the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA` Kconfig option instead of using :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_SETTINGS` stores Wi-Fi credentials in a PSA protected storage, adding more security.
+
```

Comment:
Can add a comment here about the auto connect feature.

The wifi credential shell enables auto connect that ensure that when the device reboots, or resets and there are credentials the device will automatically try to connect to the stored list of credentials.

The auto connect feature can be disabled by unsetting CONFIG_WIFI_CREDENTIALS_SHELL_AUTO_CONNECT.

Reply:
the auto connect feature only triggers a connection attempt, when rebooting the the device the device will anyhow automatically try to connect with the credentials in the stored list.